src/pages/cart/Cart.jsx

```diff
@@ -1,15 +1,14 @@
 import React from 'react'
 import { Nav, Footer } from "../../components";
-import { CartProvider } from '../../services/context/cartcontext';
 import Cartlist from './Cartlist';
 import Carttotal from './Carttotal';
 function Cart() {
     return (
         <div>
             <Nav />
```

Comment:
Instead of importing Nav component in every file you can try to keep it in `App.js`, right outside the `<Routes>` tag.